### PR TITLE
Edit Chapter 7

### PR DESCRIPTION
**Pull request by amandamuledy**
Hi! Erik has a few comments for you in his commit. Here are mine:

Line 73----Did I interpret this correctly? It wasn't quite clear to me, and I think it would have been less clear to casual readers.

Line 131--I put this here because I thought you needed an intro. It wasn't intuitive that the next thing would be a discussion of these. If you don't like it, or if I made a wrong assumption, feel free to rephrase.

Line 170--Added because I was trying to help you avoid repeated phrases and words. Please reject if it doesn't suit your purposes. 

Line 386--Making a note of the question mark so it doesn't fly under your radar!

Okay! Happy weekend. I'll review any changes you make. Meanwhile, I'll start on Chapter 8 Monday.

### CODE REVIEW

**Comment by ayende** [comment]
"the more network traffic making a decision will generate" seems off, no?

73 & 131 - looks okay

**Comment by amandamuledy** [comment]
This is one of the instances where a "that" would be valuable; it's saying "the
more network traffic that making a decision will generate." But you're
right. That's still hard to follow.

I think a better way to say it would be, "On the other hand, the more nodes
there are in your cluster, the more network traffic that will come of
making a decision."

**Comment by ayende** [comment]
Change it to be `On the other hand, the more nodes there are in your cluster, the more network traffic will be required to reach a consensus.`
